zulipterminal/ui_tools/boxes.py

```diff
@@ -163,6 +165,13 @@ def autocomplete_emojis(self, text: str, prefix_string: str
         return emoji_typeahead, emojis
 
     def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
+        if self.is_typeahead_mode:
```

Comment:
minor: I think I mentioned both `is_` and `in_`, but consider the commit text you have: "If the user is not in typeahead mode". Based on expressing that in code,  `in_` probably works better, right? You could even have `is_in_` since you have space to be verbose - assuming we leave the `not` from the commit text, to be a logical operator.